css/styles.css

```diff
@@ -428,15 +495,218 @@ footer .bottom-navigator {
   align-items: center;
 }
 
-.bottom-navigator .hr-bar {
-  width: 134px;
-  height: 5px;
-  border-radius: 10px;
-  background-color: #0d151e;
+.bottom-navigator .social-media-icons {
   margin: 15px 0;
 }
 
 .bottom-navigator .social-media-icons .link {
   color: rgba(80, 95, 121, 1);
-  margin: 15px 0;
+}
+
+.bottom-navigator .hr-bar {
+  width: 134px;
+  height: 5px;
+  border-radius: 10px;
+  background-color: #0d151e;
+  margin-bottom: 15px;
+}
+
+@media screen and (min-width: 768px) {
+  .works-section .grid-container {
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+  }
+}
+
+@media screen and (min-width: 992px) {
+  h1.section-title {
+    font-size: 72px;
+    line-height: 81px;
+  }
+
+  .section-subtitle {
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .description-text {
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .box-join-title {
+    width: 100%;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .box-join-title .hr-bar {
+    margin-top: 0;
+  }
+
+  .top-navigator {
+    padding: 16px 100px;
+  }
+
+  .top-navigator .menu-list {
+    display: flex;
+  }
+
+  .top-navigator .menu-icon {
+    display: none;
+  }
+
+  .page-section {
+    padding: 100px 100px;
+  }
+
+  .headline-section {
+    background-image: url(../images/illustration-header.svg);
```

Comment:
- [ ] Great job implementing this as a `background-image`, but right now it isn't working, as you can see here:

![image](https://user-images.githubusercontent.com/66967274/144528493-fa6fab7d-858c-4cbc-9e5b-9e74259844ca.png)